.github/workflows/ci.yaml

```diff
@@ -38,6 +38,12 @@ jobs:
         if: ${{ matrix.python-version == '3.12' }}
         with:
           path: site/
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Build Docker Image
+        uses: docker/build-push-action@v6
```

Comment:
Can we explicitly say `push: false` to make it clear to a reader? I wasn't sure whether this pushed by default or not (the name sort of implies that it does).